<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFormDebug</class>
 <widget class="QWidget" name="QFormDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>调试</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>连接设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_DeviceInfo">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>服务器IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ServerIp">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0.0.0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>23350</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_StartServer">
        <property name="text">
         <string>启动服务</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_DebugCmd">
        <property name="text">
         <string>调试命令</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>接收</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextBrowser" name="textBrowser_Receive">
         <property name="toolTip">
          <string>发送数据为蓝色字体，接收数据为绿色字体</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>参数设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox_HexDisplay">
            <property name="text">
             <string>HEX显示</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_AutoDecode">
            <property name="text">
             <string>自动解码</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ClearRec">
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>发送</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit_Send"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>参数设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_HexSend">
            <property name="text">
             <string>HEX发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_AllSend">
            <property name="text">
             <string>全部发送</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ClearSend">
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>自定义帧</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_CustomAddr">
         <property name="toolTip">
          <string>设备地址，十六进制数据，范围0x00~0xff</string>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>帧类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_CustomFrameType">
         <property name="toolTip">
          <string>数据帧地址，十进制数据，范围0~99</string>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CustomData">
         <property name="placeholderText">
          <string>HEX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_CustomSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ClearRec</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser_Receive</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ClearSend</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_Send</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
